packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -53,32 +53,64 @@ const UNIDENTIFIED_CODE = [0, 229];
  */
 export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvider>
     implements EditorPlugin {
-    private editor: IEditor;
-    private eventHandledOnKeyDown: boolean;
-    private blockSuggestions: boolean;
-    private isSuggesting: boolean;
-    private lastKnownRange: Range;
+    private editor: IEditor | null = null;
+    private core: PickerPluginCore | null = null;
+    public dataProvider: T | undefined = undefined;
+    private pickerOptions: PickerPluginOptions | undefined = undefined;
+
+    constructor(dataProvider: T | undefined, pickerOptions: PickerPluginOptions) {
+        this.dataProvider = dataProvider;
+        this.pickerOptions = pickerOptions;
+    }
 
-    // For detecting backspace in Android
-    private isPendingInputEventHandling: boolean = false;
-    private currentInputLength: number;
-    private newInputLength: number;
+    public willHandleEventExclusively(event: PluginEvent) {
```

Comment:
Originally this class doesn't have willHandleEventExclusively, so no need to add it